rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -105,9 +109,17 @@ def get_facets_sql(self, facets, table_prefix='ps'):
         facets_sql.append(filters_sql)
 
     if len(facets_sql) > 0:
-      facets_sql = 'WHERE ' + ' AND '.join(facets_sql)
+      facets_sql = ' AND '.join(facets_sql) + ' AND'
     else:
-      return ''
+      facets_sql = ''
+
+    facets_sql = 'WHERE ' + facets_sql
```

Comment:
Good point.  I used that odd sequencing because I was transforming `facets_sql` from a list to a string.  The revised code doesn't do that and uses two variables instead, simplifying the initialization.